Packs/CertStream/Playbooks/playbook-Create_list_for_PTH.yml

```diff
@@ -745,43 +1043,224 @@ tasks:
+          accessor: assistant
+          transformers:
+          - operator: StripChars
+            args:
+              chars:
+                value:
+                  simple: '{}'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -140,
+          "y": 1230
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "44":
+    id: "44"
+    taskid: 0b41e822-f88a-4f5b-81c2-6a05ce46e3d5
     type: condition
     task:
-      id: a8535e2e-b09f-47d7-8ec5-bc075377a7a7
+      id: 0b41e822-f88a-4f5b-81c2-6a05ce46e3d5
       version: -1
-      name: Manual\Xpanse permutation
-      description: ''
+      name: List exist - should overwrite?
+      description: ""
       type: condition
       iscommand: false
       brand: ""
     nexttasks:
       '#default#':
-      - "7"
-      Use LLM Permutations:
-      - "10"
+      - "9"
+      "Yes":
+      - "5"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": -530
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: List with the name ${inputs.DomainsListName} already exist, should
+          we continue and overwrite it?
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - "Yes"
+      - "No"
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "45":
+    id: "45"
+    taskid: eaf35e8c-75f8-4f6b-8c7a-95988baa2e6b
+    type: condition
+    task:
+      id: eaf35e8c-75f8-4f6b-8c7a-95988baa2e6b
+      version: -1
+      name: How would you like to create a list for PTH?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      Expanse:
+      - "15"
+      Manual:
+      - "3"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -125
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to: null
+      subject: null
+      body:
+        simple: how would you like to create the list?
+      methods: []
+      format: ""
+      bcc: null
+      cc: null
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - Expanse
+      - Manual
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "46":
```

Comment:
Please add a manual task for adding homographs after creating the list on the expanse path.